crashsimilarity/utils.py

```diff
@@ -17,6 +17,37 @@ def read_files(file_names, open_file_function=smart_open):
                 yield line if isinstance(line, str) else line.decode('utf8')
 
 
+class StackTraceProcessor(object):  # just a namespace, actually
+    @staticmethod
+    def _should_skip(stack_trace):
```

Comment:
Each commit/PR should be as clean as possible. Since they are effectively public here, let's leave them public. When/if they are going to become private, we can make them private.